src/commands/sentinel-simulate-failure.json

```diff
@@ -10,6 +10,28 @@
             "ADMIN",
             "SENTINEL",
             "ONLY_SENTINEL"
+        ],
+        "arguments": [
+            {
+                "name": "mode",
+                "type": "oneof",
+                "arguments": [
+                    {
+                        "name": "crash-after-election",
+                        "type": "pure-token"
+                    },
+                    {
+                        "name": "crash-after-promotion",
+                        "type": "pure-token"
+                    },
+                    {
+                        "name": "help",
+                        "type": "pure-token"
+                    }
+
+                ]
+            }
+            
```

Comment:
There are two extra line breaks here

Reply:
Thanks @enjoy-binbin  good catch.  I already update codes and remove extra lines, please take a look.  Thanks